### PR TITLE
Enikshay - changes in Live reports.

### PR DESCRIPTION
**Pull request by lwyszomi**
Hi @calellowitz 

I added changes in TB Notification Register live report and I added a new live report. I talk with Jordan and this changes can wait to merge until rebuilds test v5 and episode v5 data sources finished.
Please let me know if you find any issues.

Regards,
Łukasz

### CODE REVIEW

File: custom/enikshay/ucr/reports/dmc_lab_summary.json [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+  ],
+  "data_source_table": "test_2b_v5",
+  "report_id": "dmc_lab_summary",
+  "config": {
+    "description": "",
+    "title": "DMC Lab Summary",
+    "sort_expression": [],
+    "visible": true,
+    "configured_charts": [],
+    "aggregation_columns": [],
```
Comment (calellowitz) [inline custom/enikshay/ucr/reports/dmc_lab_summary.json]: is this report tested at all, my understanding was that you need something in here for a ucr to work, and properly aggregate data

**Review by calellowitz — CHANGES_REQUESTED** [review]
one question. rest looks good

File: custom/enikshay/ucr/reports/dmc_lab_summary.json (hunk 1 above)
Comment (lwyszomi) [reply custom/enikshay/ucr/reports/dmc_lab_summary.json]: @calellowitz Yes this report was tested, if we have aggregation_columns empty then in the report we have only one record, and this is expected behavior.

Comment (calellowitz) [reply custom/enikshay/ucr/reports/dmc_lab_summary.json]: sounds good

**Review by calellowitz — APPROVED** [review]

**Pull request merged**